addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -47,9 +47,8 @@
                      t-attf-style="background-image: url({{ 'data:image/png;base64,%s' % company.layout_background_image.decode('utf-8') if company.layout_background_image and company.layout_background == 'Custom' else '/base/static/img/bg_background_template.jpg' if company.layout_background == 'Geometric' else ''}});"
                      t-att-data-oe-model="o and o._name"
                      t-att-data-oe-id="o and o.id"
-                     t-att-data-oe-lang="o and o.env.context.get('lang')"
-                     t-if="not skip_headers">
-                    <table class="table-borderless">
+                     t-att-data-oe-lang="o and o.env.context.get('lang')">
+                    <table class="table-borderless" t-if="not skip_headers">
```

Comment:
Indeed, the bug occurs as follows (I write it down so I don't forget :D)
When rendering `l10n_ch.l10n_ch_qr_header`, if `skip_headers` is True, we currently loose the div with the att `data-oe-lang`, `data-oe-model` and `data-oe-id`.
Then, `_prepare_html` is not able to parse the `res_ids` and returns `[]` for the second element of the tuple returned. 
Hence, the returned dict of `_render_qweb_pdf_prepare_streams` is of the form:
`{<invoice.id>: {'stream': None, 'attachment': None}}` if we have multiple docs (`len(res_ids) > 1`).
And it fails in l10n_ch since a `stream` is expected for each invoice.
So, yep, your fix seems good :+1:

Reply:
(so it's broken since: https://github.com/odoo/odoo/commit/10f4000927e7c04892627ddeb59be71b6f2cb924)